doc/src/examples/pausable.md

```diff
@@ -0,0 +1,85 @@
+## Overview
+
+This example shows how you can reuse the implementation of
+[pausable](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/security/pausable) in `Flipper` contract to `flip` only if the contract is paused.
```

Comment:
Let's better another case with `when_not_paused`. ...in `Flipper` contract to reject `flip` when a contract is paused.